src/NetAnalyzers/Core/Microsoft.CodeQuality.Analyzers/ApiDesignGuidelines/EnumWithFlagsAttribute.Fixer.cs

```diff
@@ -75,9 +75,11 @@ private static SyntaxNode RemoveFlagsAttribute(SyntaxGenerator generator, Semant
             }
 
             AttributeData flagsAttribute = enumType.GetAttributes().First(a => Equals(a.AttributeClass, flagsAttributeType));
-            SyntaxNode attributeNode = flagsAttribute.ApplicationSyntaxReference.GetSyntax(cancellationToken);
+            SyntaxNode? attributeNode = flagsAttribute.ApplicationSyntaxReference?.GetSyntax(cancellationToken);
```

Comment:
Can this ever be null given that FlagsAttribute has `Inherited = false`: https://docs.microsoft.com/en-us/dotnet/api/system.flagsattribute?view=netcore-3.1?

Reply:
Good remark, I went defensive without asking me much question...